### DISCUSSION TITLE
ColorProfile not automatically detected

### DISCUSSION THREAD

**Discussion by adejoux** [charmbracelet/lipgloss#440] [Q&A]
**Describe the bug**

Hello, when running the programme below the Colorprofile is wrongly detected as ASCII. I need to setup it manually.

```go
func main() {

        var style = lipgloss.NewStyle().
                Bold(true).
                Foreground(lipgloss.Color("9")).
                Background(lipgloss.Color("12")).
                PaddingTop(2).
                PaddingLeft(4).
                Width(22)

        // detected colorProfile is ASCII no color in the output
        fmt.Print(style.Render("Hello, kitty"))

        renderer := lipgloss.NewRenderer(os.Stdout, termenv.WithProfile(termenv.TrueColor))

        // setting manually the Colors profile
        renderer.SetColorProfile(termenv.TrueColor)

        style2 := renderer.NewStyle().Foreground(lipgloss.Color("#FF0000")) // bright red

        // Red color is displayed properly
        fmt.Print(style2.Render("Hello, World!"))

}
```

**Setup**
Please complete the following information along with version numbers, if applicable.
 - OS macOS
 - Shell  zsh, nu Shell
 - Terminal Emulator  kitty, iterm
 - Terminal Multiplexer tmux, none
 - Locale en_US.UTF-8

**To Reproduce**
1. Run go program in the description

**Source Code**
Please include source code if needed to reproduce the behavior. 

**Expected behavior**

I expected to have the Colors without setting up anything.

**Screenshots**

![image](https://github.com/user-attachments/assets/89c7e45a-463d-4315-8d99-58504a94e97b)

**Comment by meowgorithm** [comment]
Hi! You can manually set the color profile globally with `lipgloss.SetColorProfile()`. That said, I'm curious why the detection is failing as I'm unable to reproduce the issue locally with the source code you provided.

What are `SHELL`, `TERM`, `COLORTERM`, and `CLICOLOR` set to in your environment?

**Reply by adejoux** [reply]
Hello, 

Yes I can solve it like that but it’s strange🤔

So it’s not a big issue but here the variables:

SHELL: /bin/zsh
TERM: xterm-kitty
COLORTERM: truecolor
CLICOLOR:
